src/components/utils/directory-content-utils.ts

```diff
@@ -142,7 +143,7 @@ export const getColumnsDefinition = (childrenMetadata: Record<UUID, ElementAttri
         }),
         field: 'lastModifiedBy',
         cellRenderer: UserCellRenderer,
-        flex: 1,
+        flex: 1.2,
```

Comment:
maybe keep 1 here to have same width for `owner` and `lastModifiedBy` which show th same Avatar component